s3-scan-tar/app/scanner.py

```diff
@@ -70,43 +77,53 @@ def scan_archive(blob: Blob, clamd_socket: ClamdUnixSocket, buffer_size: int, ta
 
     :raises Exception: when an unkown error is encountered
     """
-    clean, virus, skipped = 0, 0, 0
+
+    clean_files, virus_files, skipped_files = Files(), Files(), Files()
+    file_count, scanned_count = 0, 0
```

Comment:
```suggestion
    clean_files = Files()
    virus_files = Files()
    skipped_files = Files()
    file_count = 0
    scanned_count = 0
```

Er vell like greit å skrive det slik, i stedet for å tulle med en tuple som egentlig ikke blir brukt som en tuple